src/content/docs/apm/agents/ruby-agent/attributes/ruby-attribute-examples.mdx

```diff
@@ -130,27 +130,27 @@ foo, bar, request.parameters.foo, request.parameters.bar
 
 In this example, specific input keys are selected for certain output destinations and excluded from others.
 
-* The `lang.oo.python` key will be excluded only from transaction traces.
-* The `lang` and `lang.functional` keys will be excluded from all destinations.
+* The `food.fruit.banana` key will be excluded only from transaction traces.
```

Comment:
I guess the linter wants us to say something like:
Transaction traces will exclude only the `food.fruit.banana` key.